src/main.ts

```diff
@@ -325,19 +340,21 @@ export default function main() {
     feedbackPromptHandler?.(message, handlers);
   }
 
-  async function feedbackPromptEvent(
-    promptId: string,
-    event: "received" | "shown" | "dismissed",
-    userId: User["userId"],
-  ) {
+  async function feedbackPromptEvent(options: {
```

Comment:
nit: these are not technically `options` but `args`

Reply:
fair point 👍